package.json

```diff
@@ -18,7 +18,7 @@
     "eslint-plugin-react": "^7.4.0",
     "eslint-plugin-relay": "^0.0.19",
     "flow-bin": "^0.56.0",
-    "gatsby": "^2.0.0",
+    "gatsby": "^2.13.67",
```

Comment:
Old Gatsby (and scheduler) creates a version mismatch which makes it hard to use hooks!